config/locales/ru.yml

```diff
@@ -667,8 +667,6 @@ ru:
       <p>This document is CC-BY-SA. It was last updated May 31, 2013.</p>
       <p>Originally adapted from the <a href="https://github.com/discourse/discourse">Discourse privacy policy</a>.</p>
     title: Условия обслуживания и политика конфиденциальности %{instance}
-  themes:
-    default: Mastodon
```

Comment:
this was added intentionally by a human https://github.com/tootsuite/mastodon/pull/6227